### PR TITLE
Add sample with version diff application

### PR DESCRIPTION
**Pull request by e5l**

### CODE REVIEW

**Review by rsinukov — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ note on [Docs and Samples Migration](https://blog.jetbrains.com/ktor/2020/09/16/
 * [reverse-proxy-ws](reverse-proxy-ws/README.md) - A reverse proxy application using WebSockets written
   with [Ktor](https://ktor.io) and [WebSockets](https://ktor.io/docs/websocket.html).
```
Comment (andreyaksenov) [inline README.md:27]: > A reverse proxy application written using [Ktor](https://ktor.io) and [WebSockets](https://ktor.io/docs/websocket.html).

File: README.md [hunk 2]
```diff
@@ -26,6 +26,7 @@ note on [Docs and Samples Migration](https://blog.jetbrains.com/ktor/2020/09/16/
...
 * [youkube](youkube/README.md) - Video upload/view application written with [Ktor](https://ktor.io).
```
Comment (andreyaksenov) [inline README.md:28]: > A video upload/view application ...

File: README.md [hunk 3]
```diff
@@ -26,6 +26,7 @@ note on [Docs and Samples Migration](https://blog.jetbrains.com/ktor/2020/09/16/
...
+* [version-diff](version-diff/README.md) - Application showing diff between artifact in 2 versions of project.
```
Comment (andreyaksenov) [inline README.md:29]: > An application showing the difference between artifacts in two versions of a project.

File: version-diff/README.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Version diff
+
+Diff tool for maven artifacts written using [Ktor](https://ktor.io) client.
```
Comment (andreyaksenov) [inline version-diff/README.md:3]: > A diff tool for Maven artifacts written using [Ktor](https://ktor.io) client.

File: version-diff/README.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+## Running
+
+Execute this command in the sample directory:
```
Comment (andreyaksenov) [inline version-diff/README.md:7]: > Execute this command in a sample's directory:

**Review by andreyaksenov — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (e5l) [reply README.md:27]: Thanks, fixed

File: README.md (hunk 2 above)
Comment (e5l) [reply README.md:28]: Fixed

File: README.md (hunk 3 above)
Comment (e5l) [reply README.md:29]: Thank you, fixed

File: version-diff/README.md (hunk 4 above)
Comment (e5l) [reply version-diff/README.md:3]: Yep, fixed!

File: version-diff/README.md (hunk 5 above)
Comment (e5l) [reply version-diff/README.md:7]: Fixed

**Comment by e5l** [comment]
Hey @andreyaksenov, thanks for the review. All fixed! Could you take a look?

**Review by andreyaksenov — APPROVED** [review]

**Pull request merged**